packages/product-feed-heureka/src/Model/FeedItem/HeurekaFeedItemFactory.php

```diff
@@ -5,34 +5,33 @@
 namespace Shopsys\ProductFeed\HeurekaBundle\Model\FeedItem;
 
 use Shopsys\FrameworkBundle\Component\Domain\Config\DomainConfig;
+use Shopsys\FrameworkBundle\Component\LocalCache\LocalCacheFacade;
 use Shopsys\FrameworkBundle\Model\Category\CategoryFacade;
 use Shopsys\FrameworkBundle\Model\Pricing\Price;
 use Shopsys\FrameworkBundle\Model\Product\Availability\ProductAvailabilityFacade;
 use Shopsys\FrameworkBundle\Model\Product\Pricing\ProductPriceCalculationForCustomerUser;
 use Shopsys\FrameworkBundle\Model\Product\Product;
 use Shopsys\ProductFeed\HeurekaBundle\Model\HeurekaCategory\HeurekaCategoryFacade;
-use Symfony\Contracts\Service\ResetInterface;
 
-class HeurekaFeedItemFactory implements ResetInterface
+class HeurekaFeedItemFactory
 {
-    /**
-     * @var string[]|null[]
-     */
-    protected array $heurekaCategoryFullNamesCache = [];
+    protected const string HEUREKA_CATEGORY_FULL_NAMES_CACHE_NAMESPACE = 'heurekaCategoryFullNamesCache';
```

Comment:
You do not use word `Cache` in previous strings, so remove it here too.

Reply:
i dont understand what you meen